shared/menubar/index.js

```diff
@@ -187,8 +187,8 @@ class Menubar extends Component {
 
 export default connect(
   state => ({
-    username: state.config && state.config.status && state.config.status.user && state.config.status.user.username,
-    loggedIn: state.config && state.config.status && state.config.status.loggedIn,
+    username: state.config.username,
```

Comment:
@chrisnojima Hmm, this throws when I open the menu widget -- it seems that `state.config` isn't defined at the time that it opens. Is that what you expected? Should I dive into this further?